NEWS.md

```diff
@@ -4,6 +4,8 @@
 
 - Added a "Report a bug" link to `{admiralonco}` website (#211)
 
+- Added a section regarding dependencies to the `{admiralonco}` website (#223)
+
```

Comment:
do we not need to add extreme_event function changed to extreme_record( function in admiralonco) ? is this not the right place?